govcd/adminorg_ldap_test.go

```diff
@@ -62,10 +61,6 @@ func (vcd *TestVCD) Test_LDAP(check *C) {
 		check.Assert(err, IsNil)
 	}()
 
-	sleepTime := 10 * time.Second
-	fmt.Printf("# Sleeping %s to prevent 'LDAP context not initialized' errors\n", sleepTime.String())
-	time.Sleep(sleepTime)
-
```

Comment:
Remove sleep time as we know that it is not the solution for LDAP context errors